basics/installation/environments/macos-specific.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Installing an AMP stack on MacOS
+menuTitle: MacOS environment
+weight: 3
+---
+
+# Installing an AMP stack on MacOS
+
+PrestaShop is an Open-Source web application running on an *AMP stack (Apache - MySQL - PHP).
+
+In order for PrestaShop to run locally on your MacOS computer, you need to install an *AMP stack. 
+
+Several options are available, such as: 
+
+- [XAMPP](https://www.apachefriends.org/index.html)
+- [MAMP](https://www.mamp.info/en/mamp/macos/)
+- [Brew + Apache + MySQL + PHP](https://brew.sh/)
+
+## Use Docker to run PrestaShop on MacOS
+
+Another good option when it comes to run PrestaShop on MacOS, is to use `Docker`: [More informations on this dedicated page]({{< relref "8/basics/installation/environments/docker">}}).
+
+<a href="{{< relref "/8/basics/installation" >}}" class="cta cta-primary">
```

Comment:
On build blog we have this snippet:

```
{{< cta "https://github.com/PrestaShop/PrestaShop/releases/tag/8.1.0" >}}What's new in version 8.1?{{< /cta >}}
```

maybe it is available here?

Reply:
not on this instance, but we can create the shortcode in a next PR